docs/product-user-guide/src/main/asciidoc/decision-tables-defining-proc.adoc

```diff
@@ -0,0 +1,44 @@
+. Use the next four rows to define the following elements as needed (rows 14-17 in example):
++
+*<@Michael (1 of 6): I'm not clear here on which cells are required and which are optional. The drools doc says that really just the rule attribute (first bullet below) is required, like CONDITION, etc., and the others are optional. But the user needs to specify at least the second cell too, the object type, right? If all are optional, then how would the user specify the condition or action or whatever? It would just say CONDITION and then the value of the first data cell, 3 rows down, would have what? Maybe a complete standalone code snippet or something? Can you give an example of a table that leaves all the optional cells blank?>*
++
+* *Rule attributes:* Conditions, actions, or other attributes. For supported attributes, see xref:decision-tables-rule-table-entries-ref[].
+* *Object types:* (Optional) The data objects to which the rule attributes apply. Leave blank if unused. If the same object type applies to multiple columns, merge the object cells into one cell across multiple columns, instead of repeating the object type in multiple cells. When an object type is merged, all columns below the merged range will be combined into one set of constraints within a single pattern for matching a single fact at a time. When an object is repeated in separate columns, the separate columns can create different patterns, potentially matching different or identical facts.
+* *Constraints:* (Optional) Constraints on the object types, if specified. Leave blank if unused.
+* *Column label:* (Optional) Any descriptive label for the column, as a visual aid. Leave blank if unused.
++
+. After you have defined all necessary rule attributes (columns), enter values for each column as needed, row by row, to generate rules (rows 18 and 19 in example). Cells with no data are ignored (such as when a condition or action does not apply).
++
+If you need to add more rule tables to this decision table spreadsheet, skip a row after the last rule in the previous table, label another `RuleTable` cell in the same column as the previous `RuleTable` and  `RuleSet` cells, and create the new table following the same steps in this section.
++
+. Save your XLS or XLSX spreadsheet to finish.
+
+NOTE: Only the first worksheet in a spreadsheet workbook will be processed as a decision table when you upload the spreadsheet in {CENTRAL}. Each `RuleTable` name combined with the sheet name must be unique across all decision table files in the same package.  *<@Michael (2 of 6): By "sheet name", do you mean spreadsheet file name or RuleSet name?>*
```

Comment:
@sterobin It means RuleSet. The reason being is that the "package" name comes from RuleSet and then "rule" names in the resultant DRL come from the RuleTable; so I could have ```package A rule Rules_1``` from one XLS (where the RuleSet is "A" and the RuleTable is "Rules") and ```package A rule Rule_1``` from another XLS **IF** it too had a RuleSet of "A" and RuleTable of "Rules"). I believe this would cause a compilation error. You can have the same rule names in different packages or different rule names in the same package. I checked the code and neither spreadsheet file name or worksheet name are used.